dashboards/main.dashboard.lookml

```diff
@@ -165,26 +161,24 @@
     note_display: hover
     note_text: Delta compared to previous time period
     listen:
-      Time: ingestion_stats.period_filter
+      Time: ingestion_metric_with_ingestion_stats.period_filter
```

Comment:
Is should be **ingestion_metrics** if we are planning to deprecate **ingestion_stats** and **ingestion_metrics_with_ingestion_stats**.